command/main.go

```diff
@@ -23,7 +48,83 @@ func Run(args []string) int {
 			args = []string{"version"}
 			break
 		}
+
+		// Parse a given flag here, which overrides the env var
+		if strings.HasPrefix(arg, "-format=") {
```

Comment:
Haven't tested, but does this handle `--format` as well?

Reply:
Didn't even know we supported that. Will address.